Update comms guidelines (#36)
matplotlib/governance: communications_guidelines.md

# Communication Guidelines
These guidelines are applicable when acting as a representative of Matplotlib (for example at sprints or when giving official talks or tutorials) and in the following community venues managed by Matplotlib:
* library and docs: https://github.com/matplotlib/matplotlib
* forum: https://discourse.matplotlib.org/
* chat: https://matrix.to/#/#matplotlib:matrix.org
* blog: https://blog.scientific-python.org/
* active social media: 
   * https://twitter.com/matplotlib
   * https://instagram.com/matplotart/
* official accounts:
   * https://bsky.app/profile/matplotlib.bsky.social
   * https://fosstodon.org/@matplotlib
   * https://www.tiktok.com/@matplotart
   * https://www.youtube.com/matplotlib
* mailing lists:
    * https://mail.python.org/mailman/listinfo/matplotlib-announce
    * https://mail.python.org/mailman/listinfo/matplotlib-users
    * https://mail.python.org/mailman/listinfo/matplotlib-devel

## Maintenance
If you are interested in moderating the chat or forum or accessing the social media accounts:
 * matplotlib maintainer: reach out to the [community manager](people.md#deputy-project-leads) to elevate privileges.

>  * everyone else: send an email explaining why to matplotlib-social-admin@numfocus.org 

Reviewer: ```suggestion
 * everyone else: send an email describing your motivation and background matplotlib-social-admin@numfocus.org 
```
May be a bit more to the point.
